packages/core/src/components/search/search.scss

```
@@ -1,6 +1,18 @@
 @use '../../helpers';
+@use '../icon/icon';
+@use '../../mixins';
```

I think sorting is wrong, `../../` goes before `../`.

```suggestion
@use '../../mixins';
@use '../icon/icon';
```